frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -486,28 +485,39 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                             icon: <IconChat />,
                             to: urls.surveys(),
                         },
+                        featureFlags[FEATURE_FLAGS.PRODUCT_INTRO_PAGES] !== 'test' || hasOnboardedFeatureFlags
+                            ? {
+                                  identifier: Scene.EarlyAccessFeatures,
+                                  label: 'Early access features',
+                                  icon: <IconRocket />,
+                                  to: urls.earlyAccessFeatures(),
+                              }
+                            : null,
+                    ].filter(isNotNil),
+                    [
                         {
-                            identifier: Scene.DataWarehouse,
-                            label: 'Data warehouse',
+                            identifier: Scene.DataManagement,
+                            label: 'Events management',
+                            icon: <IconDatabase />,
+                            logic: isUsingSidebar ? dataManagementSidebarLogic : undefined,
+                            to: isUsingSidebar ? undefined : urls.eventDefinitions(),
+                        },
+                        {
+                            identifier: Scene.DataWarehouseSettings,
+                            label: 'Data import',
                             icon: <IconServer />,
-                            to: isUsingSidebar ? undefined : urls.dataWarehouse(),
+                            to: isUsingSidebar
+                                ? undefined
+                                : urls.dataWarehouseSettings(DataWarehouseTab.ManagedSources),
                         },
                         hasOnboardedAnyProduct
                             ? {
                                   identifier: Scene.Pipeline,
-                                  label: 'Data pipeline',
+                                  label: 'Data export',
```

Comment:
I don't think this works...

"Data export" now includes "Transformations" which are all about how the incoming events are transformed - nothing to do with exporting 🤔 

I don't mind the "Events Management" change but I'm really thinking we should stick with `Data pipelines` for now as the overall term for "data in and out". Then we just need to make some UX improvements to that section to have

`|Overview|Sources|Transformations|Destinations|` 

Happy to help put that together if we agree its the way forward